### PR TITLE
Applicant information page

### PR DESCRIPTION
**Pull request by alaminShaheen**
Added:
1. stepper context
2. translations
3. form functionality

### CODE REVIEW

File: src/components/Stepper/ApplicantInformationForm/ApplicantInformationForm.tsx [hunk 1]
```diff
@@ -1,5 +1,592 @@
...
+        I understand and agree the information submitted will be used in line
+        with our
+        <a
+          href="#"
+          className="inline-flex items-center gap-x-1 font-bold text-primary underline"
+        >
+          privacy policy
+          <ExternalLink />
+        </a>
+      </Trans>
```
Comment (AsifOni) [inline src/components/Stepper/ApplicantInformationForm/ApplicantInformationForm.tsx]: We have these checkbox content used in Signup, Confirm Identity and then Application Agreement page. Can we make this a component?

File: src/components/Stepper/ApplicantInformationForm/supplement/ApplicantInformationForm.test.tsx
Comment (AsifOni) [inline src/components/Stepper/ApplicantInformationForm/supplement/ApplicantInformationForm.test.tsx:1]: We need test cases to cover error scenario for each field

File: src/components/Stepper/StepperSidebar/StepperSidebar.tsx [hunk 2]
```diff
@@ -5,26 +5,27 @@ import { Fragment, useState } from "react";
...
       {/* LG version */}
       <ul
-        className="sticky top-[100px] hidden p-16 text-white lg:block"
+        className="sticky top-[100px] hidden px-14 py-16 text-white lg:block"
         aria-label="progress"
       >
         {steps.map((step, index) => {
           return (
-            <Fragment key={step.title}>
+            <Fragment key={step.id}>
```
Comment (AsifOni) [inline src/components/Stepper/StepperSidebar/StepperSidebar.tsx]: Let's not rely on only `id`. Add some wording so that it can be completely unique

**Review by AsifOni — CHANGES_REQUESTED** [review]
**UI Discrepencies**

1. Gap is missing

image

2. Underline missing

image

3. Button typography not matching Figma

image

4. Wider space is missing

image

5. Remove Back button - Confirmed with Business

image


6. Z-index needs to be fixed

image

7. Content and Typography design not matching Figma

image

8. Error message color and box shadow is missing as per Figma

File: src/components/Stepper/ApplicantInformationForm/supplement/ApplicantInformationForm.test.tsx
Comment (alaminShaheen) [reply src/components/Stepper/ApplicantInformationForm/supplement/ApplicantInformationForm.test.tsx:1]: Bhai the errors for each and every field is already included in the unit tests

**Review by AsifOni — CHANGES_REQUESTED** [review]
@alaminShaheen 

**UI Issue**

At below viewport, UI looks squeezed. Left container should have more width compare to Sidebar stepper (right container).

image

**Changes I made on your branch**

- Created `FormError` component and added test file
- Replaced all error message section from every component with `FormError`. 
- Added `aria-described` property to missing form elements. This was **missing** in new Form. Please keep this attribute in mind as it's absolutely important for Form to meet AODA

**Note**

Reasons for creating `FormError` component are

- It's used widely in all Form related components
- It's structure is different now since you added info icon. For future, if we need to adjust the UI again, then we will have only one place to change

For similar kind of situation, please think it thoroughly before working on development and create component model accordingly

**Review by AsifOni — APPROVED** [review]

**Pull request merged**